### PR TITLE
CLDC-1228: Handles household charges

### PR DESCRIPTION
**Pull request by smeny**

### CODE REVIEW

**Comment by baarkerlounger** [comment]
As discussed:

- County field will be removed
- Location#location_code field will store the LA
- Creating a location will do the postcode API call to look up LA

Ideally we want to have all fields defined on scheme or location to be pointed to by the case log rather than duplicated by the case log.

File: app/services/imports/case_logs_import_service.rb [hunk 1]
```diff
@@ -590,5 +601,49 @@ def apply_household_consistency!(attributes)
...
+          attributes["pscharge"].nil? &&
+          attributes["supcharg"].nil?
+        attributes["brent"] ||= BigDecimal("0.0")
+        attributes["scharge"] ||= BigDecimal("0.0")
+        attributes["pscharge"] ||= BigDecimal("0.0")
+        attributes["supcharg"] ||= BigDecimal("0.0")
+      end
+    end
+
+    def clear_household_charges(attributes)
```
Comment (smeny) [inline app/services/imports/case_logs_import_service.rb]: Delete this method

File: spec/services/imports/case_logs_import_service_spec.rb [hunk 2]
```diff
@@ -203,5 +215,17 @@ def open_file(directory, filename)
         end
       end
     end
+
+    context "and this is a supported housing log" do
```
Comment (smeny) [inline spec/services/imports/case_logs_import_service_spec.rb:219]: expect no warning

File: app/services/postcode_service.rb [hunk 3]
```diff
@@ -1,4 +1,25 @@
 class PostcodeService
+  def initialize
+    @pio = Postcodes::IO.new
+  end
+
+  def infer_la(postcode)
```
Comment (baarkerlounger) [inline app/services/postcode_service.rb:6]: Love this! (Refactoring it out into the service)

File: app/models/case_log.rb [hunk 4]
```diff
@@ -102,6 +102,22 @@ def self.editable_fields
     attribute_names - AUTOGENERATED_FIELDS
   end
 
+  def la
```
Comment (baarkerlounger) [inline app/models/case_log.rb:105]: Nice!

**Review by baarkerlounger — APPROVED** [review]

**Pull request merged**